app/views/admin/loans/print_memo.slim

```diff
@@ -0,0 +1,22 @@
+- content_for(:title, @loan.name)
+
+section data-container="loan"
+  header
+    .btn.btn-default data-action="print" Print
+    h1 Loan Memo
```

Comment:
Ensure items use translations. Same with loan details and loan memo print views.